### ISSUE TITLE
'shell' subcommand doesn't print current version

### ISSUE THREAD

**Issue by eugenesvk** [dyuri/xontrib-langenv#17]
`pyenv shell` is supposed to `echo "$PYENV_VERSION"` for bash to print the actual version (if the version is defined), but in this plugin nothing is printed

It might be xonsh-related, not specific to this plugin since
  -  `source-bash --suppress-skip-message -n "echo 1"` doesn't print `1` (asked this question in xonsh discussions https://github.com/xonsh/xonsh/discussions/5019)

**Comment by dyuri** [comment]
Yep, this seems to be `source-bash` related, it might need a new flag to show the output, so let's keep this open.
